### PR TITLE
Agges home

### PR DESCRIPTION
**Pull request by alfredamboka**
Updated  content. add social media links.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
         "navcol"
     ],
     "workbench.colorCustomizations": {
-        "titleBar.activeBackground": "#ae10fd"
+        "titleBar.activeBackground": "#f39b09"
```
Comment (OkomoJacob) [inline .vscode/settings.json]: - Get the AGGES team to agree on uniform color codes

File: css/styles.min.css [hunk 2]
```diff
@@ -167,7 +167,82 @@
     font-size: 42px;
 }
```
Comment (OkomoJacob) [inline css/styles.min.css:168]: - Same applies to custom fonts(Families and sizes)

File: pages/Events.html [hunk 3]
```diff
@@ -25,7 +25,7 @@
 
 <body style="background: rgb(238,244,247);">
     <nav class="navbar navbar-light navbar-expand-md" style="background: #01011f;">
-        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/logo_blue.png" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
+        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/aggesLogo.jpg" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
```
Comment (OkomoJacob) [inline pages/Events.html]: - Replace logo.jpg with logo.png,. Prefer pngs to jpgs

File: pages/about.html [hunk 4]
```diff
@@ -16,7 +16,7 @@
 
 <body style="background: rgb(238,244,247);">
     <nav class="navbar navbar-light navbar-expand-md" style="background: #01011f ;">
-        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/logo_blue.png" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
+        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/aggesLogo.jpg" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
```
Comment (OkomoJacob) [inline pages/about.html]: Same here

File: pages/about.html [hunk 5]
```diff
@@ -50,7 +50,7 @@ <h2 class="divider-style"><span><strong>About Us</strong></span></h2>
                     <p style="color: rgb(27,103,215);"><strong>AGGES: Who are we?</strong></p>
                 </div>
                 <div>
-                    <p style="padding-left: 5px;"><strong>Association of Geospatial Engineering Students</strong>&nbsp; is one of among the clubs at <strong>JKUAT.&nbsp; </strong>The&nbsp; club consist of like minded passionate Geomatic Engineering, Geospatial Engineering, and GIS
+                    <p style="padding-left: 5px;"><strong>Association of Geospatial Engineering Students</strong>&nbsp; is among the clubs at <strong>JKUAT.&nbsp; </strong>The&nbsp; club consist of like minded passionate Geomatic Engineering, Geospatial Engineering, and GIS
```
Comment (OkomoJacob) [inline pages/about.html:53]: - The AGGES Projects team will guide in wordings and chose of phrases

File: pages/members.html [hunk 6]
```diff
@@ -16,7 +16,7 @@
 
 <body style="background: rgb(238,244,247);">
     <nav class="navbar navbar-light navbar-expand-md" style="background: #01011f;">
-        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/logo_blue.png" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
+        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/aggesLogo.jpg" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
```
Comment (OkomoJacob) [inline pages/members.html]: - Same here

**Review by OkomoJacob — APPROVED** [review]
- Kindly update the changes with the AGGES Projects team

**Comment by OkomoJacob** [comment]
- Will fix the resolve/conflicts into main

**Review by OkomoJacob — COMMENTED** [review]
- Done

File: pages/about.html (hunk 5 above)
Comment (alfredamboka) [reply pages/about.html:53]: Noted. will review this by the executive team and the dev team that we will form for maintenance

File: pages/members.html (hunk 6 above)
Comment (alfredamboka) [reply pages/members.html]: Done. updated logo

File: pages/members.html [hunk 7]
```diff
@@ -16,7 +16,7 @@
 
 <body style="background: rgb(238,244,247);">
     <nav class="navbar navbar-light navbar-expand-md" style="background: #01011f;">
-        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/logo_blue.png" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
+        <div class="container-fluid"><a class="navbar-brand" href="../index.html" style="color: var(--cyan);"><img src="../img/aggesLogo.png" style="width:50px; height:25px"></a><button data-toggle="collapse" class="navbar-toggler" data-target="#navcol-1" style="color: var(--red);background: #c60e0e;"><span class="sr-only">Toggle navigation</span><span class="navbar-toggler-icon" style="color: rgba(152,9,9,0.5);"></span></button>
```
Comment (OkomoJacob) [inline pages/members.html:19]: - Noted, better updates

File: pages/archive.html [hunk 8]
```diff
@@ -7,24 +7,23 @@
...
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300|Poiret+One|Raleway:400,800|Rokkitt:400,700">
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700">
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300|Poiret+One|Raleway:400,800|Rokkitt:400,700">
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
         <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.12.0/css/all.css">
-        <link rel="stylesheet" href="https://cdn.datatables.net/buttons/1.6.5/css/buttons.dataTables.min.css"
+        <link rel="stylesheet" href="https://cdn.datatables.net/buttons/1.6.5/css/buttons.dataTables.min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.0.0-beta2/css/bootstrap.min.css">
```
Comment (OkomoJacob) [inline pages/archive.html:19]: - Better views now 👍🏽

**Review by OkomoJacob — COMMENTED** [review]
- Reviews confirmed

**Pull request merged**